src/js/chat.js

```diff
@@ -827,3 +834,11 @@ function initSock() {
 
     };
 }
+
+window.addEventListener("message", screenshotHook, false);
+
+function screenshotHook(event) {
+    if (event.origin === "http://localhost:9000")
```

Comment:
This should instead be a regex that either allows any localhost or any subdomain of eternagame.org or eternadev.org

Reply:
To be clear, `/https?:\/\/((localhost)|(*.eternagame.org)|(*.eternadev.org)):*/`